main/main.go

```diff
@@ -138,7 +136,9 @@ func main() {
 
 	mux := &sync.Mutex{}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	cfg := ctrl.GetConfigOrDie()
+	cfg.Wrap(common.RetryMiddleware(log)) // allow all client calls to be retriable
```

Comment:
Should this be added to dynamicClient and clientset as well?

Reply:
Added